extensions/2.0/Vendor/EXT_feature_metadata/schema/featureCollection.schema.json

```diff
@@ -0,0 +1,40 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema",
+    "title": "Feature collection",
+    "type": "object",
+    "description": "A feature collection designated by a feature class and property values stored in arrays.",
+    "allOf": [ { "$ref": "glTFProperty.schema.json" } ],
+    "properties": {
+        "name": {
```

Comment:
We should also add a description property to feature collections like what classes and properties have